dbt_subprojects/dex/models/trades/base/platforms/uniswap_v4_base_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+    schema = 'uniswap_v4_base'
+    , alias = 'base_trades'
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['tx_hash', 'evt_index']
+    , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v4_trades(
+        blockchain = 'base'
+        , project = 'uniswap'
+        , version = '4'
+        , Pair_evt_Swap = source('uniswap_v4_base', 'PoolManager_evt_Swap') 
+        , Factory_evt_PoolCreated = source('uniswap_v4_base', 'PoolManager_evt_Initialize')
```

Comment:
```suggestion
        , PoolManager_evt_Swap = source('uniswap_v4_base', 'PoolManager_evt_Swap') 
        , PoolManager_evt_Initialize = source('uniswap_v4_base', 'PoolManager_evt_Initialize')
```

Reply:
to hosuke's point, since naming standards changed across versions, makes sense to keep param names clean